holochain-overlay/default.nix

```
@@ -9,6 +9,20 @@ self: super: {
     ln -fs ${pkg}/bin/${pkg.pname}-${pkg.version} $out/bin/${alias}
   '';
 
+  # Helper function to select the appropriate architecture and construct the config
+  selectArchConfig = { system ? builtins.currentSystem, version, linux_x64, darwin_x64 ? null, darwin_aarch64 ? null, windows_x64 ? null }:
```

Is this `builtins.currentSystem` the only part making flakes not work?  Since it is just a default can it just be removed?

```suggestion
  selectArchConfig = { system, version, linux_x64, darwin_x64 ? null, darwin_aarch64 ? null, windows_x64 ? null }:
```

---

Actually, it looks like it can be replaced with `super.system`

```suggestion
  selectArchConfig = { system ? super.system, version, linux_x64, darwin_x64 ? null, darwin_aarch64 ? null, windows_x64 ? null }:
```